src/fontra/views/editor/editor.js

```diff
@@ -1411,7 +1411,15 @@ export class EditorController {
       { title: "Cancel", isCancelButton: true },
       { title: "Add", isDefaultButton: true, result: "ok", disabled: true },
     ]);
-    let addToAllSources = false;
+    const addToAllSourcesLocalStorageKey = "fontra-add-the-component-to-all-sources";
+    let addToAllSources = localStorage.getItem(addToAllSourcesLocalStorageKey);
```

Comment:
Let's change the check to `=== "true"` when reading, and write "false" or "true".

```suggestion
    let addToAllSources = localStorage.getItem(addToAllSourcesLocalStorageKey) === "true";
```